src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working");
```

Comment:
Similarly, the message 'Excavator stopped working' should be more informative. It should indicate that the Excavator has stopped its operations and perhaps the reason or state after stopping.